Formula/geant4.rb

```diff
@@ -0,0 +1,101 @@
+class Geant4 < Formula
+  desc "Simulation toolkit for particle transport through matter"
+  homepage "http://geant4.cern.ch"
+  url "http://geant4.cern.ch/support/source/geant4.10.04.p02.tar.gz"
+  version "10.4.2"
+  sha256 "2cac09e799f2eb609a7f1e4082d3d9d3d4d9e1930a8c4f9ecdad72aad35cdf10"
+
+  option "with-g3tog4", "Use G3toG4 Library"
```

Comment:
Please remove all options, choosing a "best use case" scenario for most users. No experimental code (unless upstream enables it by default).

Reply:
Okay. I don't think anyone uses G3toG4 anymore, though I'm not aware of a downside to including it, so I'll remove it. I'll remove the other options. I'll leave the Qt GUI in as optional.